com.google.AndroidStudio.json

```diff
@@ -12,6 +12,8 @@
         "--share=ipc",
         "--share=network",
         "--device=all",
+        "--env=ANDROID_EMULATOR_USE_SYSTEM_LIBS=1",
```

Comment:
Shouldn't this be `0` and it just uses the embedded one already? Or is this working around a bug.

Reply:
Hi, this is an attempt to overcome a 

> Cannot initialize GL driver

Error

Reply:
I can split this out to a separate PR